packages/-ember-data/package.json

```diff
@@ -73,13 +73,14 @@
     "ember-qunit": "*",
     "ember-qunit-assert-helpers": "*",
     "ember-resolver": "*",
-    "ember-source": "*",
+    "ember-source": "~3.4.0",
     "ember-source-channel-url": "*",
     "ember-try": "*",
     "github": "*",
     "json-typescript": "*",
     "loader.js": "*",
-    "typescript": "*"
+    "typescript": "*",
+    "@ember/jquery": "^0.6.0"
```

Comment:
same with this, if we need to specify it with a `*` that's fine but shouldn't be a specific version. for dev-deps the specific version should live in the root package.json

Reply:
same as above.